cfgov/ask_cfpb/models/pages.py

```diff
@@ -586,22 +548,72 @@ class AnswerPage(CFGOVPage):
         on_delete=models.SET_NULL,
         related_name='redirected_pages',
         help_text="Choose another Answer to redirect this page to")
+    redirect_to_page = models.ForeignKey(
+        'self',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='redirect_to_pages',
+        help_text="Choose another AnswerPage to redirect this page to")
+    featured = models.BooleanField(
+        default=False,
+        help_text=(
+            "Check to make this one of two featured answers "
+            "on the landing page."))
+    featured_rank = models.IntegerField(blank=True, null=True)
+    category = models.ManyToManyField(
+        'Category',
+        blank=True,
+        help_text=(
+            "Categorize this answer. "
+            "Avoid putting into more than one category."))
+    subcategory = models.ManyToManyField(
+        'SubCategory',
+        blank=True,
+        help_text=(
+            "Choose only subcategories that belong "
+            "to one of the categories checked above."))
+    search_tags = models.CharField(
+        max_length=1000,
+        blank=True,
+        help_text="Search words or phrases, separated by commas")
+    related_resource = models.ForeignKey(
+        RelatedResource,
+        blank=True,
+        null=True
+    )
+    related_questions = models.ManyToManyField(
+        'self',
+        symmetrical=False,
+        blank=True,
+        related_name='related_question',
+        help_text='Maximum of 3')
+    answer_id = models.IntegerField(default=0)
 
     content = StreamField([
         ('feedback', v1_blocks.Feedback()),
     ], blank=True)
 
     content_panels = CFGOVPage.content_panels + [
         MultiFieldPanel([
-            FieldRowPanel([FieldPanel('last_edited')])
-        ],
-            heading="Visible time stamp"),
-        FieldPanel('question'),
-        FieldPanel('statement'),
-        FieldPanel('snippet'),
-        FieldPanel('answer'),
-        FieldPanel('search_tags'),
-        FieldPanel('redirect_to'),
+            FieldPanel('question', classname="title"),
```

Comment:
I think applying `classname="title"` to questions and statements make them difficult to edit because the text is too big to fit in the form window.

Reply:
@higs4281 @virginiacc Is this something we should take care of / remove now, or will your enhancements take care of this kind of thing?

Reply:
I left those classname specifiers off in the initial migration, because it can make long questions hard to read. I don't think we should re-introduce them.